package.json

```diff
@@ -18,6 +18,18 @@
       "title": "Works for Private Browsing",
       "type": "bool",
       "value": true
+    },
+    {
+      "name": "maxOpenInNextWindow",
+      "title": "Open in next available window",
```

Comment:
If there wasn't another window already open, I could agree, @cheeaun ! ;-)

---

What is the issue 19, @rindeal ?